cypress/integration/Index/index.spec.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference types="cypress" />
```

Comment:
I guess we can remove the `.keep` file on the same level as we already have this `Index` folder

Reply:
Removed `.keep` https://github.com/Lahphim/react-surveyor-challenge/pull/47/commits/ce5a176e03fbdf0ea70a9eb1cd1c0a885920e4d7